charts/enterprise-logs/templates/admin-api/deployment-admin-api.yaml

```diff
@@ -42,8 +42,10 @@ spec:
       priorityClassName: {{ .Values.adminApi.priorityClassName }}
       {{- end }}
       securityContext:
-        {{- toYaml .Values.adminApi.securityContext | nindent 8 }}
+        fsGroup: 10001
```

Comment:
If I understand this correctly, this affects all containers in the Pod but I guess minio is fine with this as its user can still read the configuration that is mounted in. We need the Pod level SecurityContext to support the volumeMounts in the AdminAPI container because the Container SecurityContext doesn't affect the Pod Volumes.

Reply:
Right. `fsGroup` is only supported on the Pod spec, but not on container level. Since minio runs as root, this is ok.